gobblin-runtime/src/main/java/gobblin/runtime/AbstractJobLauncher.java

```diff
@@ -243,6 +248,20 @@ public void launchJob(JobListener jobListener) throws JobException {
       jobState.setStartTime(startTime);
       jobState.setState(JobState.RunningState.RUNNING);
 
+      // Set up task staging dir and task output dir
```

Comment:
Not sure why a comment was not showing up here. But I think this part is config-related and should better be put into `JobContext`.